src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,264 @@
+
+
+The [Web Animations API](https://www.w3.org/TR/web-animations-1/) is a tool that enables developers to write [imperative animations with JavaScript](https://www.youtube.com/watch?v=WaNoqBAp8NI). It was written to underpin both CSS animation and transition implementations and enable future effects to be developed, as well as existing effects to be composed and timed.
+
+While it has a long history, originally launching in Chrome 36, the latest release, Chrome 84, brings us a slew of previously unsupported features. 
+
+<figure class="w-figure">
+  <img class="w-screenshot" src='./waapi-timeline.png' alt="The Web Animations API first hit Chrome in version 36, July of 2014. Now the spec is going to be complete, in version 84, launching July 2020.">
+  <figcaption class="w-figcaption">
+    The long history of the Web Animations API.
+  </figcaption>
+</figure>
+
+## Getting started
+
+Creating an animation via the Web Animations API should feel very familiar if you've used `@keyframe` rules. First you'll need to create a Keyframe Object. What might look like [this](https://codepen.io/una/pen/RwWMvPw) in CSS:
+
+```css
+@keyframes openAnimation {
+  0% {
+    transform: scale(0);
+  }
+  100% {
+    transform: scale(1);
+  }
+}
+```
+
+would look like [this](https://codepen.io/una/pen/abvYXJX) in JavaScript:
+
+```js
+const openAnimation = [
+  { transform: 'scale(0)' }, 
+  { transform: 'scale(1)' }, 
+];
+```
+
+Where you set parameters for animation in CSS:
+
+```css
+.modal {
+  animation: openAnimation 1s 1 ease-in;
+}
+```
+
+you would set in JS:
+
+```js
+document.querySelector('.modal').animate(
+    openAnimation, {
+      duration: 1000, // 1s
+      iterations: 1, // single iteration
+      easing: 'ease-in' // easing function
+    }
+);
+```
+
+The amount of code is about the same, but with JavaScript, you get a couple of superpowers that you don't have with CSS alone. This includes the ability to sequence effects, and an increased control of their play states.
+
+{% Aside %}
+  Hyphenated property names become camel case when used in keyframes (e.g. `"background-color"` to `"backgroundColor"`)
+{% endAside %}
+
+### Getting and setting animations beyond `element.animate()`
+
+However, with the update, the Web Animations API is no longer restricted to animations created via `element.animate()`. We can manipulate CSS animations and transitions as well.
+
+`getAnimations()` is a method that returns all animations on an element regardless of whether it was created via `element.animate()` or via CSS rules (CSS animation or transition). Here is an example of what this looks like:
+
+
+<div class="glitch-embed-wrap" style="height: 420px; width: 100%;">
+  <iframe
+    src="https://glitch.com/embed/#!/embed/waapi-getanimations?path=style.css&previewSize=100"
+    title="waapi-getanimations on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
```

Comment:
```suggestion
```

It looks like you used the embed code from Glitch (rather than [our code in the handbook](https://web.dev/handbook/web-dev-components/#glitches)). Please delete all of these `allow` attributes in your embedded Glitches unless you actually need these permissions. It seems like a potential attack surface that I'd like to avoid.